### ISSUE TITLE
No package matching 'postgrey' found available, installed or updated

### ISSUE THREAD

**Issue by darnel** [nestihacky/ansible-role-mailserver#23]
Error in issue title appears when running mail playbook.

I think that `postgrey` is not in default repo. Only clue I found is that in previous CentOS potgrey was installed from RPMForge, which is dead now and is replaced by EPEL.

[Adding EPEL repo solution](http://www.tecmint.com/how-to-enable-epel-repository-for-rhel-centos-6-5/) makes `postgrey` searchable by yum.

Sorry for misleading information, I'm not regular CentOS user...

**Comment by mmilata** [comment]
Yeah, we probably need to add a dependency on https://github.com/nestihacky/ansible-role-epel
